packages/transport/src/car/request.js

```diff
@@ -0,0 +1,69 @@
+import * as API from '@ucanto/interface'
+import { CAR } from '@ucanto/core'
+import { Delegation } from '@ucanto/core'
+
+export { CAR as codec }
+
+const HEADERS = Object.freeze({
+  'content-type': 'application/car',
```

Comment:
seems like this media type should be defined in `@ipld/js-car` or otherwise stored in a constant somewhere